banzai_floyds_ui/gui/utils/plot_utils.py

```diff
@@ -47,11 +47,34 @@ def extraction_region_traces(order_polynomial, center_polynomial, width_polynomi
     extract_center += order_polynomial(x)
     extract_center += center_delta
     extract_high = extract_center + extract_upper * extract_sigma
-    extract_low = extract_center - extract_lower * extract_sigma
+    extract_low = extract_center + extract_lower * extract_sigma
     background_upper_start = extract_center + bkg_right_inner * extract_sigma
-    background_lower_start = extract_center - bkg_left_inner * extract_sigma
+    background_lower_start = extract_center + bkg_left_inner * extract_sigma
     # Let's start with 10 sigma for now as the edge rather than using the whole slit
-    background_lower_end = extract_center - bkg_left_outer * extract_sigma
+    background_lower_end = extract_center + bkg_left_outer * extract_sigma
     background_upper_end = extract_center + bkg_right_outer * extract_sigma
     return x, [extract_center, extract_low, extract_high, background_lower_start, background_upper_start,
                background_lower_end, background_upper_end]
+
+
+def plot_extracted_data(frame_data):
+    figure_data = []
+    plot_column = {2: 1, 1: 2}
+    for order in [2, 1]:
+        where_order = frame_data['order'] == order
+        top_row_axis = '' if plot_column[order] == 1 else plot_column[order]
+        figure_data.append(
+            dict(type='scatter', x=frame_data['wavelength'][where_order], y=frame_data['flux'][where_order],
+                 line=dict(color='#023858'), mode='lines', xaxis=f'x{top_row_axis}', yaxis=f'y{top_row_axis}')
```

Comment:
Maybe nice to define a constant with a meaningful for this color so it could be changed more easily and readers know roughly what color to expect.